api/Window.json

```diff
@@ -50,6 +50,92 @@
           "deprecated": false
         }
       },
+      "cancelAnimationFrame": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/cancelAnimationFrame",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "21",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "21",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "23"
+              },
+              {
+                "version_added": "11",
+                "version_removed": "23",
+                "prefix": "moz"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "23"
+              },
+              {
+                "version_added": true,
```

Comment:
This should be 14, as Firefox for Android [11](https://www.mozilla.org/firefox/android/11.0beta/releasenotes/), [12](https://www.mozilla.org/firefox/android/12.0beta/releasenotes/) and [13](https://www.mozilla.org/firefox/android/13.0beta/releasenotes/) only made it to beta due to instabilities or something.